RELEASING.md

```diff
@@ -9,30 +9,36 @@ bundle exec pod trunk info BlueprintUI
 bundle exec pod trunk info BlueprintUICommonControls
 ```
 
-See [the CocoaPods documentation for pod trunk](https://guides.cocoapods.org/making/getting-setup-with-trunk) for more information about setting up credentials on your device.
+See [the CocoaPods documentation for pod trunk](https://guides.cocoapods.org/making/getting-setup-with-trunk) for more information about setting up credentials on your device. If you need to be added as an owner, ping in #blueprint on Slack (Square only).
 
-1. Make sure you're on the `master` branch
+1. Make sure you're on the `master` branch, and `git pull` to get the latest commits.
 
-1. Make sure that the library version in both `BlueprintUI.podspec` and `BlueprintUICommonControls.podspec` is correct (it should match the version number that you are about to release).
+1. Create a branch off `master` to update the version numbers and `Podfile.lock`. An example name would be `kve/release-0.9`.
+
+1. Update the library version in both `BlueprintUI.podspec` and `BlueprintUICommonControls.podspec` if it has not already been updated (it should match the version number that you are about to release).
 
 1. Run `bundle exec pod install` to update the `Podfile.lock` with the new versions.
 
-1. Create a commit containing the podspec version bumps and the `Podfile.lock` update, and tag the commit with the version number.
+1. Commit the podspec version bumps and the `Podfile.lock` update.
    ```bash
-   git commit -am "Releasing 0.1.0."
-   git tag 0.1.0
+   git commit -am "Bumping versions to 0.1.0."
    ```
 
-1. Push your changes
+1. Push your branch and open a PR into `master`.
+
+1. Once the PR is merged, switch back to `master` and tag the release:
    ```bash
-   git push origin master && git push origin 0.1.0
+   git checkout master
+   git pull
+   git tag 0.1.0
```

Comment:
This should probably reference the exact commit of the merge now, in case HEAD is not the PR that was just merged, e.g. `git tag 0.1.0 <sha-from-merge>`

Reply:
Ah true